script/Arcueid/Day 8/08_00_ARC08_8.txt

```diff
@@ -622,6 +622,7 @@ But it's abundantly clear that this man is anything but.
 [sha:a335a764e74c9d130c6a03319a42c5a7ce147101]{
 -- Page 517, Offset 21065.
 -- 「“―――ジャマ、ヲ”」
+// add a space between g and e?
```

Comment:
hm yeah I guess it should either be more consistent or more fucked up. No real preference which direction.

Reply:
Going with the former then